docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [ ] Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
 
 ### Point Clouds
 
-- Support visualization of time-based differencing of large scale point clouds.
+- [ ] Support visualization of time-based differencing of large scale point clouds.
 
 ### Reality Data Sources
 
-- Implement a grid of linear transformations for large reality data.
+- [ ] Implement a grid of linear transformations for large reality data.
 
 ### Geolocated Photos and Panoramas
 
-- Improve server-side extraction of tags from photos and panoramas.
-- Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
+- [ ] Improve server-side extraction of tags from photos and panoramas.
+- [ ] Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
 
 ### Geolocation (general)
 
-- Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
-- Implement frontend reprojection when possible based on orbit gt implementation.
-- Implement backend of new geo coord service reprojection for complex gcs.
-- Obtain geoid separation from new geo coord service.
+- [X] Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
+- [ ] Implement frontend reprojection when possible based on orbit gt implementation.
+- [ ] Implement backend of new geo coord service reprojection for complex gcs.
+- [ ] Obtain geoid separation from new geo coord service.
 
 ### iModel Transformations
 
-- Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
+- [ ] Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
 
 ### iTwin.js based connector framework
 
-- Support for creating multi-process connectors using iTwin.js backends. One process links with source application api to read the source application files. It then communicates with another iTwin.js process via IPC to update the iModel.
+- [ ] Support for creating multi-process connectors using iTwin.js backends. One process links with source application api to read the source application files. It then communicates with another iTwin.js process via IPC to update the iModel.
 
 ### iModel Editing applications
 
-- Support interactive iModel creation, editing, and augmenting on desktops via Electron.
+- [X] Support interactive iModel creation, editing, and augmenting on desktops via Electron.
 
 ### Mobile Device Support
 
-- Minimize memory, power, and bandwidth requirements for mobile browsers.
-- Improve usability for touch and small screen form factors.
-- Support both *completely offline* and *occasionally connected* tablets and phones (iOS and Android).
+- [ ] Minimize memory, power, and bandwidth requirements for mobile browsers.
```

Comment:
@toddsouthenbentley @tcobbs-bentley please mark any of these items as completed.

Reply:
I'm not sure about minimizing memory, power and bandwidth above.
The offline and occasionally connected use cases below are not quite done. (It is asking for the access token any time the frontend communicates with the fully local backend, and it should not be doing this. I'm talking to Affan and Raman about that now.)